Gemfile

```diff
@@ -14,4 +14,11 @@ end
 # your gem to rubygems.org.
 
 # Load Gemfile with dependencies from manageiq
-eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+manageiq_gemfile = File.expand_path("spec/manageiq/Gemfile", __dir__)
+if File.exist?(manageiq_gemfile)
+  eval_gemfile(manageiq_gemfile)
+else
+  puts "ERROR: The ManageIQ application must be present in spec/manageiq."
+  puts "  run `bin/setup` to clone it from GitHub or symlink it from local source."
```

Comment:
@Fryguy I slightly tweaked your wording to indicate that there is `bin/setup` to do the job